spring-data-neo4j/src/main/java/org/springframework/data/neo4j/repository/support/Neo4jRepositoryFactoryBean.java

```diff
@@ -66,18 +67,12 @@ public void afterPropertiesSet() {
 		super.afterPropertiesSet();
 	}
 
+	/*
+	 * (non-Javadoc)
+	 * @see org.springframework.data.repository.core.support.TransactionalRepositoryFactoryBeanSupport#doCreateRepositoryFactory()
+	 */
 	@Override
 	protected RepositoryFactorySupport doCreateRepositoryFactory() {
-		return createRepositoryFactory(session);
-	}
-
-	/**
-	 * Returns a {@link RepositoryFactorySupport}.
-	 *
-	 * @param session
-	 * @return
-	 */
-	protected RepositoryFactorySupport createRepositoryFactory(Session session) {
```

Comment:
How about deprecating the method first? `Neo4jRepositoryFactoryBean` isn't the typical user-facing API, yet there might be someone that has customized `createRepositoryFactory(…)`.